test/e2e/framework/pod/wait.go

```diff
@@ -312,6 +313,15 @@ func WaitForMatchPodsCondition(c clientset.Interface, opts metav1.ListOptions, d
 	return fmt.Errorf("gave up waiting for matching pods to be '%s' after %v", desc, timeout)
 }
 
+// GetPodDescription returns a string representing a pod in a consistent human readable format,
+// with pod UID as part of the string.
+// This should match the format defined in kubernetes/pkg/kubelet/util/format.
+func GetPodDescription(podName, podNamespace string, podUID types.UID) string {
```

Comment:
If there are no other references, I think this function can be set to private.
SGTM